src/app/public/components/public-archive-web-links/public-archive-web-links.component.ts

```diff
@@ -9,11 +9,31 @@ export class PublicArchiveWebLinksComponent implements OnInit {
   componentName: string = 'public-archive-web-links';
 
   @Input() description: string = '';
-  @Input() email: string = '';
-  @Input() website: string = '';
-  @Input() facebook: string = '';
+  @Input() emails: string[] = [];
+  @Input() websites: string[] = [];
+
 
   constructor() {}
 
   ngOnInit(): void {}
+
+  navigate(url: string): void {
+    if (!url.startsWith('http://') && !url.startsWith('https://')) {
+        url = 'https://' + url;
+    }
+    window.open(url, '_blank');
```

Comment:
Technically you're [not supposed to open links in a new tab](https://www.w3.org/TR/WCAG20-TECHS/G200.html) by default. I don't know if we have a standard practice for this in the codebase so far, but I know our users generally want the back button to work. Do you know what we do elsewhere, @meisekimiu ? If there isn't a clear pattern, let's open these in the same tab.